web/src/model.rs

```diff
@@ -0,0 +1,188 @@
+use serde::Deserialize;
+use yew::prelude::{html, Component, ComponentLink, Html, NodeRef, ShouldRender};
+use yew::services::fetch::FetchTask;
+
+use crate::message::Msg;
+use crate::updater;
+
+pub struct Model {
+    pub audience_input_ref: NodeRef,
+    pub permission_input_ref: NodeRef,
+    pub audience: Option<String>,
+    pub permissions: Vec<String>,
+    pub token: Option<Jwt>,
+    pub link: ComponentLink<Self>,
+    pub task: Option<FetchTask>,
+}
+
+impl Component for Model {
+    type Message = Msg;
+
+    type Properties = ();
+
+    fn create(_props: Self::Properties, link: ComponentLink<Self>) -> Self {
+        Self {
+            link,
+            audience_input_ref: NodeRef::default(),
+            permission_input_ref: NodeRef::default(),
+            task: None,
+            audience: None,
+            permissions: vec![],
+            token: None,
+        }
+    }
+
+    fn update(&mut self, msg: Self::Message) -> ShouldRender {
+        updater::update(self, msg)
+    }
+
+    fn change(&mut self, _props: Self::Properties) -> ShouldRender {
+        false
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <div class="container main spacing-v-xl">
+                <div class="form-grid">
+                    <div class="form-grid__row form-grid__row--small">
+                        <legend class="form-legend">
+                            <span class="form-legend__title">{"LOCALAUTH0"}</span>
+                            <span class="form-legend__addon">
+                                <img
+                                    src="assets/static/media/localauth0.png"
+                                    width="80"
+                                    height="80"
+                                    alt="Localauth0 logo"
+                                />
+                            </span>
+                        </legend>
```

Comment:
Manca un div con la colonna prima del `legend` e l'`addon` andrebbe usata sempre prima del titolo
```suggestion
                        <div class="form-grid__row__column">
                            <legend class="form-legend">
                                <span class="form-legend__addon">
                                    <img
                                        src="assets/static/media/localauth0.png"
                                        width="80"
                                        height="80"
                                        alt="Localauth0 logo"
                                    />
                                </span>
                                <span class="form-legend__title">{"LOCALAUTH0"}</span>
                            </legend>
                        </div>
```